src/components/organisms/UserProfileModal/UserProfileModal.scss

```diff
@@ -35,8 +35,9 @@ $modal-max-width: 540px;
   }
 
   .private-chat-container {
-    height: 300px;
-    overflow-wrap: break-word;
```

Comment:
Can we re-add this one? It makes sure we wrap by split-
ting words.

Reply:
I just added it back.